packages/youbet-sdk/src/modules/contractModule.ts

```diff
@@ -32,7 +32,7 @@ export class ContractModule {
       const hexChainId = '0x' + Number(this.sdk.sdkOptions.networkOptions.chainId).toString(16);
       await provider.send('wallet_addEthereumChain', [{
         chainId: hexChainId,
-        chainName: 'OpenCampus-Testnet',
+        chainName: this.sdk.sdkOptions.chainName || 'Unknown Chain',
```

Comment:
I suggest if chainName is empty just crash.

Reply:
`chainName` is not a required parameter, and it may not be called in actual scenarios. Of course, you can make it mandatory, but it may not have much practical significance; it is generally used when adding a new network for testing purposes.